bridge/contracts/AllowTokens.sol

```diff
@@ -68,7 +68,7 @@ contract AllowTokens is Initializable, UpgradableOwnable, UpgradableSecondary, I
 
     function _calcMaxWithdraw(TokenInfo memory info, Limits memory limits) private view returns (uint256 maxWithdraw) {
         // solium-disable-next-line security/no-block-members
-        if (block.timestamp > info.lastDay + 24 hours) {
+        if (block.timestamp > info.lastDay + 24 hours) { // solhint-disable-line not-rely-on-time
```

Comment:
these directives are for the compiler not to cry, right?

Reply:
Actually is for the lint warnings